docs/group-applications-workflow.md

```diff
@@ -0,0 +1,37 @@
+1. Admin creates an event with slug. Admin remembers the slug.
+
+1. Admin bulk uploads people sent by the partner and assigns them to the training. This requires to use CSV file with five columns:
+
+    - personal name,
+    - family name,
+    - email address,
+    - the training slug (all rows with the same value) and
+    - role ('learner' in all rows).
+
+1. Admin goes to "trainees" page and filter by training. That way, Admin can list exactly the same set of people that he uploaded. In the same view, Admin can filter by whether an training request was sent or not.
```

Comment:
Is "filter by whether we have a training request in the database" new functionality? Or does it already exist?